frontend/src/pages/home/modelCatalog/ModelCatalogHint.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import {
+  Button,
+  Card,
+  CardBody,
+  CardHeader,
+  Content,
+  Grid,
+  GridItem,
+} from '@patternfly/react-core';
+import { TimesIcon } from '@patternfly/react-icons';
+import modelCatalogImg from '~/images/homepage-model-catalog.svg';
+
+const ModelCatalogHint: React.FC<{ isHidden: boolean; setHidden: React.FC }> = ({
+  isHidden,
+  setHidden,
+}) => {
+  if (isHidden) {
+    return null;
+  }
+
+  const hintActions = (
+    <Button
+      icon={<TimesIcon />}
+      data-testid="model-catalog-hint-close"
+      aria-label="model catalog hint close"
+      variant="plain"
+      onClick={setHidden}
+    />
+  );
+
+  const gridImgStyle: React.CSSProperties = {
+    height: '100%',
+    backgroundSize: '90%',
+    backgroundRepeat: 'no-repeat',
+    backgroundImage: `url(${modelCatalogImg})`,
+    backgroundPosition: 'left',
+    marginLeft: '-2em',
+  };
+
+  const cardBodyStyle: React.CSSProperties = {
+    paddingLeft: 0,
+    paddingTop: 0,
+    paddingRight: '4em',
+  };
```

Comment:
For consistency, I would suggest you use inline styles directly instead of creating a variable here unless the styles need to be reused by another component. In that way, we might create a CSS class.